drivers/SmartThings/zwave-sensor/src/test/test_enerwave_motion_sensor.lua

```diff
@@ -81,6 +81,12 @@ test.register_coroutine_test(
           mock_sensor,
           Battery:Get({})
       ))
+      --This extra battery get is because of apiv6 default handler override sub driver
+      --It was left in so when we remove that subdriver, a new bug isn't created
```

Comment:
We could specifically avoid these if we wanted to, but I thought that it opened us up to extra bugs. Like the multisensor case, I will try to exclude this device from the bugfix with fingerprints.